torch_xla/csrc/ops/ops.cpp

```diff
@@ -692,7 +692,22 @@ torch::lazy::NodePtr Gelu(const torch::lazy::Value& input) {
   auto lower_fn = [](const XlaNode& node,
                      LoweringContext* loctx) -> XlaOpVector {
     xla::XlaOp xla_input = loctx->GetOutputOp(node.operand(0));
-    return node.ReturnOp(BuildGelu(xla_input), loctx);
+
+    // Building composite computation.
+    const std::string name = "composite.gelu";
+    const std::string attr = "{approximate = \"none\"}";
```

Comment:
This is a dummy str for testing purpose?

Reply:
This is a real op attribution for `GELU`: https://pytorch.org/docs/stable/generated/torch.nn.GELU.html#torch.nn.GELU

The available value of `approximate` is `none` or `tanh`. The lowering process checks this attribution and decides the sub lower function [here](https://github.com/pytorch/xla/blob/49de8bcb8fbe6af064f8a5a525ef13b9d55cf6c3/torch_xla/csrc/tensor_methods.cpp#L1706-L1712). As my changes are in the sub lower function, I manually set this attribution.

It's a common process for composite op which has **attributions** (defined as non-tensor inputs for composite op, e.g. `dim` for `Softmax`).

Reply:
Maybe I can get the attribution from XlaOp instead of manually setting strings, I will try.